docs/guides/sap/odata-connector.md

```diff
@@ -0,0 +1,170 @@
+---
+id: odata-connector
```

Comment:
If these are indeed alpha, could add these Connectors to https://docs.camunda.io/docs/next/components/connectors/out-of-the-box-connectors/available-connectors-overview/.

Reply:
yes - but they need to be run in "hybrid" mode, on the SAP BTP of the customer. We don't run them in SaaS or on any SM k8s

Reply:
Note to myself to add something on this once merged into the main PR and I can properly route to these links.